src/main/scala/cromwell/Boot.scala

```diff
@@ -0,0 +1,63 @@
+package cromwell
+
+import java.io.File
+import java.nio.file.{Files, Paths}
+
+import akka.actor.{Props, ActorSystem}
+import akka.io.IO
+import akka.pattern.ask
+import akka.util.Timeout
+import com.typesafe.config.{Config, ConfigFactory}
+import com.wordnik.swagger.model.ApiInfo
+import cromwell.engine.WorkflowManagerActor
+import cromwell.webservice._
+import spray.can.Http
+
+import scala.concurrent.duration._
+import scala.reflect.runtime.universe._
+import scala.util.{Failure, Success}
+
+
+object Boot extends App {
```

Comment:
If you rebase on top of the develop branch there's a CromwellServer object where the spray initialization would go. Not critical at this moment but we'll need to merge it together at some point.